LayoutTests/imported/w3c/web-platform-tests/css/css-text/letter-spacing/letter-spacing-animating-font-size.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>percentage letter-spacing animates when animating font-size</title>
+<link rel="author" title="Sam Weinig" href="mailto:sam@webkit.org">
+<link rel='help' href='https://www.w3.org/TR/css-text-3/#letter-spacing'>
+<link rel='match' href='reference/letter-spacing-animating-font-sizing-ref.html'>
+<style>
+  #target {
+    letter-spacing: 100%;
+  }
+</style>
+</head>
+<body>
+<div>The letters below should be separated by 20 pixels.</div> 
+<div id="target">ABCD</div>
+
+<script>
+  const target = document.querySelector('#target');
+
+  let animation = target.animate(
+    [
+      { fontSize: '0px' },
+      { fontSize: '40px' }
+    ],
```

Comment:
That's perfectly fine, but a more idiomatic way to write this would be:

```suggestion
    { fontSize: ['0px', '40px'] },
```